packages/proxy/lib/http/response-middleware.ts

```diff
@@ -420,7 +419,7 @@ const CopyCookiesFromIncomingRes: ResponseMiddleware = async function () {
     return this.next()
   }
 
-  // Cross-origin Cookie Handling
+  // First/Third Party Cookie Handling
```

Comment:
What does first party and third party mean in this context?

Reply:
resolving this at is no longer applicable.